### ISSUE TITLE
[Quest][Old Hillsbrad Foothills] Internment buildings not lighting on fire

### ISSUE THREAD

**Issue by trrrboi** [dalaranwow/dalaran-wow#5871]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

During Escape from Durnholde, after setting the Incendiary Bombs inside each barrel in each of the 5 buildings, Lieutenant Drake spawns, but the buildings do not light on fire, and the quest does not complete. Killing him does not set the buildings on fire or complete the quest either. Resetting the dungeon multiple times has not resulted in a fix.

**Expected behaviour**: Tell us what should happen instead

The buildings should light on fire and the NPCs should flee the courtyard, or the quest should at least be mark as complete.

**Steps to reproduce the problem**:

1. Enter Escape from Durnholde, receive Pack of Incendiary Bombs
2. Use bombs on barrels in interment buildings in the courtyard
3. Nothing happens
![image](https://user-images.githubusercontent.com/63116196/78383144-2506c900-7595-11ea-9bd8-1084b5c486d7.png)

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/63116196/78383166-2cc66d80-7595-11ea-9273-15c74a3ec7ee.png)
![image](https://user-images.githubusercontent.com/63116196/78383188-38b22f80-7595-11ea-94f3-94fcba540d7e.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills** 

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
